cmd/timescaledb-tune/main.go

```diff
@@ -45,9 +45,16 @@ func init() {
 	flag.BoolVar(&f.UseColor, "color", true, "Use color in output (works best on dark terminals)")
 	flag.BoolVar(&f.DryRun, "dry-run", false, "Whether to just show the changes without overwriting the configuration file")
 	flag.BoolVar(&f.Restore, "restore", false, "Whether to restore a previously made conf file backup")
+	flag.StringVar(&f.Profile, "profile", "", "a specific \"mode\" for tailoring recommendations to a special workload type. If blank or unspecified, a default is used. The TSTUNE_PROFILE environment variable overrides this flag. Valid values: \"promscale\"")
```

Comment:
IMO, we want to avoid having to publish a whole new docker image for promscale. We'd like to continue to piggyback on the timescaledb ha image. However, that image will include a call to timescaledb-tune with no argument specified for the profile. i.e. it will use the default profile. In order to use this docker image for promscale purposes, you'd want to use an environment variable passed to `docker run` to cause timescaledb-tune to run in the promscale profile mode.

Reply:
AFAIK, that means the environment variable will need to override the cmd line arg.

Reply:
Can you think of a better way?

Reply:
hmm since ts-tune will be run without specific profile in the docker image the environment variable only needs to override the default profile